README.Rmd

```diff
@@ -121,12 +121,19 @@ terra::plotRGB(res, 4, 3, 2, scale = max(landsat[["maxValues"]]))
 
 ## Authorization and publication
 
-### Authorization for non-public data access
+Authorization is not required for reading data from any public ArcGIS Online or ArcGIS Enterprise service. 
 
-Authorization is required for users who wish to access data that is in a non-public ArGIS Online (AGOL) organization, in an ArcGIS Enterprise portal, or wish to upload data to or modify data in AGOL.
+Workflows that require authorization using an ArcGIS Online or ArcGIS Enterprise account include:
```

Comment:
Wondering if the same idea might be useful here, removing "ArcGIS Online or ArcGIS Enterprise" and leave it vague because this could include "Platform" and "Hub" (i assume?) as well

Reply:
Good point- removing specific product mentions seems better here!